evasquez26/angular4-startup
# S2-ONE-123 Request role teacher

## Story / Task

see [ONE-123](https://bixlabs.atlassian.net/browse/ONE-123)


## What this PR does?

Adds functionality to request teacher role


### Before 

You can not request the role teacher

### After

You can now request the role: teacher. You can do it from the list of roles within an organization, or select a different organization

### How to test / reproduce

- Login to the app
- Go to Profile -> Roles
- You should see the list of roles by organization
- You should be able to select add role and send a role request from an organization
- You should be able to select new role from outside an organization

### Related PRs
N/A

## Migrations

NO

Thanks!
